modules/apache-airflow/templates/values.yaml

```diff
@@ -681,14 +681,13 @@ workers:
     #   cpu: 100m
     #   memory: 128Mi
 
-
-  resources: {}
-  #  limits:
-  #   cpu: 100m
-  #   memory: 128Mi
-  #  requests:
-  #   cpu: 100m
-  #   memory: 128Mi
+  resources:
+    limits:
```

Comment:
Let's leave limits on all of these commented out. I'd rather we only set the requests